users/migrations/0009_profile_highest_education.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 2.2.3 on 2019-08-05 11:50
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [("users", "0008_profile_fields_optional")]
+
+    operations = [
+        migrations.AddField(
+            model_name="profile",
+            name="highest_education",
+            field=models.IntegerField(
+                blank=True,
+                choices=[
+                    (None, "----"),
+                    (1, "Doctorate"),
+                    (2, "Professional"),
+                    (3, "Masters"),
+                    (4, "Bachelor"),
+                    (5, "Associate"),
+                    (6, "High School"),
+                    (0, "Other"),
```

Comment:
Question for @pdpinch, related to [your comment](https://github.com/mitodl/mitxpro/issues/831#issuecomment-518187353) for the issue: do you want the education values stored in the database for this field (and in hubspot) to be numerical (0-6), or the actual words like "Doctorate" and "Masters"?

Reply:
Meanwhile could you please review the other changes, if @pdpinch  wants this to change, I will change. For consistency, I used the same approach that we had used for other choice fields.